sdk/core/core/src/az_http_response.c

```diff
@@ -297,7 +297,7 @@ static az_span _az_http_response_get_remaining(az_http_response const* response)
   return az_span_slice_to_end(response->_internal.http_response, response->_internal.written);
 }
 
-AZ_NODISCARD az_result az_http_response_write_span(az_http_response* response, az_span source)
+AZ_NODISCARD az_result az_http_response_append(az_http_response* response, az_span source)
```

Comment:
Looks like this method has no unit tests.

Reply:
good catch!  Added tests just now. Thank you